src/transforms/pipelines/mod.rs

```diff
@@ -131,8 +138,11 @@ impl EventTypeConfig {
     }
 
     /// Expands a group of pipelines into a series of pipelines.
-    /// They will then be expanded into a series of transforms.
-    fn serial(&self) -> Box<dyn TransformConfig> {
+    /// If the serial mode is used, they will then be expanded into a series of transforms and an
+    /// aggregating transform will be added.
```

Comment:
It's not entirely clear to me why we add an aggregation at the end of the series. Previously I thought we'd said we just name the final stage as the name of the pipeline. Is this because this is at a higher level and we need to fan-in from pipelines of the two different types of events?